src/Symfony/Component/Mime/Tests/Header/UnstructuredHeaderTest.php

```diff
@@ -244,4 +244,20 @@ public function testGetBody()
         $header = new UnstructuredHeader('Subject', 'test');
         $this->assertEquals('test', $header->getBody());
     }
+
+    public function testInReplyToIsUnstructuredHeader()
```

Comment:
`testInReplyToAcceptsNonIdentifierValues` would be a better test name (same for the test about `References`, as this is what we care about (the fact that it is an UnstructuredHeader is an implementation detail in that regard)